astropy/time/tests/test_basic.py

```diff
@@ -1136,3 +1137,18 @@ def test_sum_is_equivalent():
 
     assert t0 == t1
     assert (t0 + 1 * u.second) == (t1 + 1 * u.second)
+
+
+def test_string_valued_columns():
```

Comment:
Should probably be `test_bytes_valued_inputs`.

Since the implementation and scope now includes `bytes` inputs that are *not* `Column`, that should be tested.